pkgs/top-level/python-aliases.nix

```diff
@@ -268,6 +268,7 @@ mapAliases ({
   imdbpy = throw "imdbpy has been renamed to cinemagoer"; # added 2022-08-08
   image-match = throw "image-match has been removed because it is no longer maintained"; # added 2023-06-10
   imgaug = throw "imgaug has been removed as it is no longer maintained"; # added 2023-07-10
+  imgtool = throw "imgtool has been promoted to a top-level attribute name: `mcuboot-imgtool`"; # added 2024-10-09
```

Comment:
```suggestion
  imgtool = super.pkgs.mcuboot-imgtool; # added 2024-10-09
```
does that work as expected?

Reply:
Nope, it doesn't:

```
imgtool should use `buildPythonPackage` or `toPythonModule` if it is to be part of the Python packages set.
```

Which is expected since this is no longer a Python package (and should never have been).